File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/asynchronousNotificationsofChanges/model/ServiceGuideChangeNotification.java
package org.ngbp.jsonrpc4jtestharness.rpc.asynchronousNotificationsofChanges.model;

import org.ngbp.jsonrpc4jtestharness.rpc.receiverQueryApi.model.ServiceGuideUrls;

import java.util.List;

public class ServiceGuideChangeNotification {
    public String msgType;
    public List<ServiceGuideUrls> urlList;
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/asynchronousNotificationsofChanges/model/ServiceGuideChangeNotification.kt
package org.ngbp.jsonrpc4jtestharness.rpc.asynchronousNotificationsofChanges.model

import org.ngbp.jsonrpc4jtestharness.rpc.receiverQueryApi.model.ServiceGuideUrls

data class ServiceGuideChangeNotification (
    var msgType: String? = null,
    var urlList: MutableList<ServiceGuideUrls?>? = null
)